cvmfs/util/posix.cc

```diff
@@ -715,6 +715,16 @@ bool FileExists(const std::string &path) {
           S_ISREG(info.st_mode));
 }
 
+bool PathExists(const std::string &path) {
+  platform_stat64 info;
+  return ((platform_lstat(path.c_str(), &info) == 0));
+}
+
+bool RegularExists(const std::string &path) {
+  platform_stat64 info;
+  return (S_ISREG(info.st_mode));
```

Comment:
`info` is not initialized here.  In any case, I don't think we actually need `RegularExists`.  We should be able to work only with `PathExists`